cypher/cypher-docs/src/docs/dev/general/query-transactions.asciidoc

```diff
@@ -16,14 +21,13 @@ While it is not possible to run a Cypher query outside a transaction, it is poss
 
 Note that the transaction will hold the changes in memory until the whole query, or whole set of queries, has finished executing.
 A query that makes a large number of updates will consequently use large amounts of memory.
-For memory configuration in Neo4j, see the <<operations-manual#memory-configuration, Neo4j Operations Manual>>.
+For memory configuration in Neo4j, see the <<operations-manual#memory-configuration, Neo4j Operations Manual -> Memory configuration>>.
```

Comment:
We originally left out the detail in the link name because it was implied by context. Adding it add redundancy to the sentence. But perhaps you did this because of some other rule you have for inter-document link formatting?

Reply:
The latter is the standard way of writing cross links and I feel consistency is more important than the fact that it's a bit redundant given the context.